src/Pages/Recommend/RecommendComponent/RecommendModalTapComponent/RecommendModalSecond.js

```diff
@@ -0,0 +1,104 @@
+import React, { Component } from "react";
+import styled from "styled-components";
+
+export class RecommendModalSecond extends Component {
+  constructor() {
+    super();
+    this.state = {
+      email: "@"
+    };
+  }
+  handleEmailInput = e => {
+    this.setState({
+      [e.target.name]: e.target.value
+    });
+  };
+
+  render() {
+    return (
+      <ModalSceond
+        className={
+          this.state.email.includes("@") ? "emailMessage" : "warningMessage"
+        }
+      >
+        <div>
+          <input
+            name="email"
+            type="text"
+            placeholder="이메일을 입력해 주세요."
+            onChange={this.handleEmailInput}
+            className="warningMessage"
+          />
+        </div>
+        <div>
+          <span
+            className={
+              this.state.email.includes("@") ? "emailMessage" : "warningMessage"
+            }
```

Comment:
```this.state.email.includes("@")를 계속 사용하시니까 변수로 관리하시는 게 더 깔끔할 것 같네요!
```suggestion
  render() {
    const isValid = this.state.email.includes("@")
    return (  
      <ModalSceond
        className={
          isValid ? "emailMessage" : "warningMessage"
        }
      >
        <div>
          <input
            name="email"
            type="text"
            placeholder="이메일을 입력해 주세요."
            onChange={this.handleEmailInput}
            className="warningMessage"
          />
        </div>
        <div>
          <span
            className={
             isValid ? "emailMessage" : "warningMessage"
            }
```